### PR TITLE
rename config, update gulp, add mit license

### PR DESCRIPTION
**Pull request by dmassiani**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -182,10 +182,9 @@ gulp.task('deps', ['html:dist'], function () {
     // Rewrites occurences of filenames which have been renamed by rev
     .pipe($.revReplace())
     // Minifies html
-    .pipe($.if('*.html', $.minifyHtml({
-      empty: true,
-      spare: true,
-      quotes: true
+    .pipe($.if('*.html', htmlmin({
```
Comment (vhpoet) [inline gulpfile.js]: You haven't defined `htmlmin`.

Comment (dmassiani) [inline gulpfile.js]: Did you mean by $. ?

Comment (vhpoet) [inline gulpfile.js]: yep, should be `$.htmlmin`

**Comment by vhpoet** [comment]
Thank you @dmassiani !

**Pull request merged**